backend/src/data_entry/repository.rs

```diff
@@ -20,27 +20,34 @@ impl PollingStationDataEntries {
     pub async fn get_row(
         &self,
         polling_station_id: u32,
-    ) -> Result<PollingStationDataEntry, sqlx::Error> {
```

Comment:
Please keep `sqlx::Error` just like in https://github.com/kiesraad/abacus/pull/1622#discussion_r2159166570